hw_19.09.23/task3.py

```diff
@@ -0,0 +1,66 @@
+class Product:
+    def __init__(self, type_product, name, price):
+        self.type_product = type_product
+        self.name = name
+        self.price = price
+
+    def __repr__(self):
+        return f"{self.type_product}:{self.name}:{self.price}"
+
+
+class ProductStore:
+
+    def __init__(self):
+        self.storage = {}
+        self.income = 0
+
+    def add(self, product, amount):
+        product.price *= 1.3
+        self.storage[product.name] = {
+            "type": product.type_product,
+            "price": product.price,
+            "amount": amount
+        }
+
+    def set_discount(self, identifier, percent, identifier_type='name'):
+        for product, amount in self.storage.items():
+            if product.type_product == identifier_type or product.name == identifier:
+                product.price *= (1 - percent / 100)
+
+            print("There is no such product")
```

Comment:
Should have been in `else` block or outside the block after a `return` as I can guess